mentat/config.py

```diff
@@ -35,22 +35,52 @@ class Config:
     # Model specific settings
     model: str = attr.field(
         default="gpt-4-1106-preview",
-        metadata={"auto_completions": list(known_models.keys())},
+        metadata={
+            "auto_completions": list(known_models.keys()),
+            "description": "Choose from one of the supported models: " + ",".join(
+                list(known_models.keys())
+            ),
```

Comment:
I'd prefer not to put the models in the description. This description will make sense in the command line but not in the sphinx documentation. I prefer "The LLM used to respond to user requests and make edits."

Reply:
Description sounds good. Models, I feel, can be in the description since they are hard-coded in the code rather than being driven by a configuration.

Reply:
I agree with @jakethekoenig on this one; I would prefer not to put the list of models in the description. If argument parser has a way to add autocomplete, I would be ok with that.